src/commcare_cloud/manage_commcare_cloud/changelog.md.j2

```diff
@@ -1,18 +1,24 @@
-# About this changelog
+# {{ ordinal }}. {{ changelog_entry.title }}
 
-The following changes to `commcare-cloud` that require your attention,
-newest first.
+**Date:** {{ changelog_entry.date }}
 
-Changes that will require an action from anyone choosing
-to update on or after the date listed will be maked "_action required_".
-Those which are worth taking a look at but which may or may not require
-an action on your part will be marked "_action optional_".
+**Optional per env:** {% if changelog_entry.optional_per_env %}Yes{% else %}No{% endif %}
 
+{% if changelog_entry.max_commcare_version -%}
+## CommCare Version Dependency
+CommCare versions beyond this commit require this change to function correctly:
+[{{ changelog_entry.max_commcare_version[:8] }}](https://github.com/dimagi/commcare-hq/commit/{{ changelog_entry.max_commcare_version }})
+{% endif %}
 
-## Changelog
-{%- for changelog_entry in changelog_contents %}
+{%- if changelog_entry.min_commcare_version -%}
+## CommCare Version Dependency
```

Comment:
nit (i think): If both max and min are defined, you'd get this same heading twice. I think I'd rather Always have this heading, and then have a phrase like "This change is not known to be dependent on any particular version of CommCare HQ."

Reply:
Never mind - I see you already did that.